lib/modules/platform/gitea/index.spec.ts

```diff
@@ -231,6 +231,21 @@ describe('modules/platform/gitea/index', () => {
       ).toMatchSnapshot();
     });
 
+    it('should support custom endpoint including api path', async () => {
+      helper.getCurrentUser.mockResolvedValueOnce(mockUser);
+
+      await expect(
+        gitea.initPlatform({
+          token: 'some-token',
+          endpoint: 'https://gitea.renovatebot.com/api/v1',
+        })
+      ).resolves.toEqual(
+        expect.objectContaining({
+          endpoint: 'https://gitea.renovatebot.com/',
+        })
+      );
```

Comment:
```suggestion
      expect(
       await gitea.initPlatform({
          token: 'some-token',
          endpoint: 'https://gitea.renovatebot.com/api/v1',
        })
      ).toMatchObject(
        {
          endpoint: 'https://gitea.renovatebot.com/',
        }
      );
```
simplify, can't we use equal match here like the other test? there should only be two properties?

Reply:
We could either use toEqual() and copy the [`gitAuthor = '${user.full_name || user.username} <${user.email}>';` line](https://github.com/renovatebot/renovate/blob/ef470f7f923a4c03e4c6204094dd508f0a4eef96/lib/modules/platform/gitea/index.ts#L195) from index.ts into the test, or alternatively use toMatchObject() in the way you suggest. I'd prefer the second option because we do not copy code.